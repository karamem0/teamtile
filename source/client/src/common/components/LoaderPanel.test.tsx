//
// Copyright (c) 2021-2024 karamem0
//
// This software is released under the MIT License.
//
// https://github.com/karamem0/teamtile/blob/main/LICENSE
//

import React from 'react';

import Presenter from './LoaderPanel.presenter';
import { render } from '@testing-library/react';

it('should create a shapshot', () => {
  const params = {};
  const { asFragment } = render(
    <Presenter {...params} />
  );
  expect(asFragment()).toMatchSnapshot();
});
