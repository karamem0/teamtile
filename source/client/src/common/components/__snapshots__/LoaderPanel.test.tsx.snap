// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  min-height: calc(100vh - 0.5rem);
}

@media (width >= 600px) {
  .emotion-0 {
    min-height: calc(100vh - 2rem);
  }
}

<div
    class="emotion-0"
  >
    <div
      class="fui-Spinner r82apo5"
      role="progressbar"
    >
      <span
        class="fui-Spinner__spinner rvgcg50 ___1xxu43e_eahmz60 f1d2rq10 f1szoe96 fb52u90"
      >
        <span
          class="fui-Spinner__spinnerTail rxov3xa"
        />
      </span>
    </div>
  </div>
</DocumentFragment>
`;
